### DISCUSSION TITLE
forward whole links without slugs - Question

### DISCUSSION THREAD

**Discussion by Skyfay** [shlinkio/discussions#81] [Help wanted]
I'm looking for a service that can forward entire URLs.
So as an example: I have a discord server and a subdomain.
Now I want to forward it as dc.example.com to discord.gg/xyxyxy
Now I have several subdomains and I would like to forward them all in exactly the same way.
I know the BASE_URL_REDIRECT_TO, ENV.
Now the question is does this work with multiple domains in one container or do I need a separate container for each domain?

**Comment by acelaya** [comment]
Shlink supports multiple domains https://shlink.io/documentation/advanced/multiple-domains/

You can even use different redirects for each one of them https://shlink.io/documentation/advanced/special-redirects/

You can make sure the same path is forwarded by using the {ORIGINAL_PATH} placeholder.

**Comment by Skyfay** [comment]
Can you make an example?

**Comment by acelaya** [comment]
An example how?

**Comment by Skyfay** [comment]
So if I now have two domains dc.example.com and dc.test.com, how should I configure it in docker?

**Comment by acelaya** [comment]
You can find everything explained in the links I shared earlier, and everything regarding the docker image is here https://shlink.io/documentation/install-docker-image/

**Comment by Skyfay** [comment]
i already got the docker setup but im not sure if i have to add something with Placeholders, Per-domain-redirects etc.
Somehow the instructions don't match my specific case

**Comment by acelaya** [comment]
Shlink is not strictly designed for this specific use case, but it can cover it. Everything is explained in those links, I don't know how to provide any further information.

**Comment by acelaya** [comment]
I'll convert this into a discussion, in case someone is trying something similar and can provide some input.
